AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Campaigns/LockUnlockCampaignCommandHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using AllReady.Models;
+using MediatR;
+using System.Linq;
+
+namespace AllReady.Areas.Admin.Features.Campaigns
+{
+    public class LockUnlockCampaignCommandHandler : RequestHandler<LockUnlockCampaignCommand>
+    {
+        private AllReadyContext _context;
+
+        public LockUnlockCampaignCommandHandler(AllReadyContext context)
+        {
+            _context = context;
+
+        }
+        protected override void HandleCore(LockUnlockCampaignCommand message)
+        {
+            var campaign = 
+                _context.Campaigns.SingleOrDefault(c => c.Id == message.CampaignId);
+
+            if (campaign != null)
+            {
+                if (campaign.Locked)
```

Comment:
@mheggeseth Sorry Mike. I'm a little confused by this suggestion. Can you expand?